src/main/java/nl/cherement/jak/config/SecurityConstants.java

```diff
@@ -12,31 +12,30 @@
 
 public class SecurityConstants {
 
+    private final Properties properties;
+
```

Comment:
rare witregel

Reply:
Is om de static en andere variables van elkaar te onderscheiden.